stdlib/public/TensorFlow/Execution.swift

```diff
@@ -37,10 +37,31 @@ public enum DeviceKind {
 @inline(never)
```

Comment:
`@inline(never)` on device placement APIs are a GPE artifact and can be removed.